client/src/components/WishList/WishList.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from "react";
+import "../Cart/cart-style.css";
+import axios from "../../utils/ajax-helper";
+import { getJWT } from "../../utils/jwt";
+import SimpleNavBar from "../SimpleNavBar/SimpleNavBar";
+import { Link } from "react-router-dom";
+import ProductTable from "../Cart/ProductTable";
+export default function Cart() {
+  // get items for the server and store in state and update in cart component
+  const [wishListItems, setWishListItems] = React.useState([]);
+  const [errorMsg, setErrorMsg] = React.useState(null);
+
+  const getWishlistItems = () => {
+    axios
+      .get("/wishlist", {
+        headers: {
+          Authorization: `Bearer ${getJWT()}`,
+        },
+      })
+      .then((res) => {
+        console.log(res.data);
+        setWishListItems(res.data);
+      })
+      .catch((err) => {
+        setErrorMsg("Sorry! Something went wrong. Please Try again " + err);
+      });
+  };
+
+  useEffect(() => {
+    getWishlistItems();
+  }, []);
+  // delete item from cart
+  const deleteItem = (id) => {
+    if (window.confirm("Are you sure you want to delete this item?")) {
+      axios
+        .delete(`/wishlist/${id}`, {
+          headers: {
```

Comment:
this headers object is repeated in many files in multiple places right? can we move this to a separate helper file and import it everywhere as a constant?